src/actions/pause.js

```diff
@@ -272,14 +280,18 @@ function deleteExpression(expression: Expression) {
  * @param {number} selectedFrameId
  * @static
  */
-function evaluateExpressions(selectedFrameId: frameIdType) {
+function evaluateExpressions(frameId: frameIdType) {
   return async function({ dispatch, getState, client }: ThunkArgs) {
     for (let expression of getExpressions(getState())) {
+      if (!expression.input) {
+        continue;
+      }
+
```

Comment:
since we are no longer adding empty expressions, this check might not be needed here anymore, maybe?